secv_guis/base_widgets.py

```diff
@@ -291,20 +294,35 @@ def add_item(self, name, rgba, slider_visible=True, activate=False):
         sl.setMinimum(0)
         sl.setMaximum(self.thresh_num_steps)
         sl.setSingleStep(1)
-        sl.setValue(sl.maximum() // 2)
+        sl.setValue(sl.maximum())
+        #
+        sl2 = QtWidgets.QSlider(None, orientation=QtCore.Qt.Horizontal)
+        lbl2 = QtWidgets.QLabel()
```

Comment:
Why do we have now 2 sliders and labels? Could you explain this added feature e.g. in the docstring? I can't quite follow from the code alone

Reply:
It is more general. In the case you use i.e. HSV images to filter your image for color or if your filter is not clean enough, you don't have to preprocess the images further but can select one/multiple specific percentile between the highest and the lowest percentile.